.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "acquire-common"]
 	path = acquire-common
 	url = ../acquire-common.git
+[submodule "aws-sdk-cpp"]
+	path = aws-sdk-cpp
+	url = git@github.com:aws/aws-sdk-cpp.git
```

Comment:
Should this be a specific revision?

Reply:
I suppose we could put it there. It's already a specific rev that's committed to this repo, but it would be clearer to developers what tag that points to. I went down all kinds of routes, including removing the submodule, to get a good build/install solution, so if you had that in there and I took it out, I apologize.